block/retriever.go

```diff
@@ -43,7 +43,9 @@ func (m *Manager) syncToTargetHeight(targetHeight uint64) error {
 
 		err = m.syncFromDABatch()
 		if err != nil {
-			return fmt.Errorf("process next DA batch: %w", err)
+			m.logger.Error("DA syncing", "height", currH, "err", err)
+			continue
+			// return fmt.Errorf("process next DA batch: %w", err)
```

Comment:
remove grey code

Reply:
i actually changed it back, so it returns error in case it fails after all retries.